src/Commands/ServiceBus/Queue/QueueDetailsCommand.cs

```diff
@@ -26,7 +26,7 @@ Get details about a Service Bus queue. Returns queue properties and runtime info
         lock duration, max message size, queue size, creation date, status, current message counts, etc.
 
         Required arguments:
-        - namespace: Service Bus namespace name. (This is usually in the form <namespace>.servicebus.windows.net)
+        - namespace: The fully qualified Service Bus namespace host name. (This is usually in the form <namespace>.servicebus.windows.net)
```

Comment:
super-nit: `namespace` and `<namespace>.servicebus.windows.net` still could be interpreted as namespace only, no `servicebus.windows.net`, maybe

```suggestion
        - namespace: The fully qualified Service Bus namespace host name. (This is usually in the form <service-bus-resource-name>.servicebus.windows.net)
```

?

Reply:
sorry, I didn't notice the PR is in MCP, thought it's in Java and was surprised with unfamiliar doc strings :)